client/src/questions-n-answers/tracker.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import {interactions} from '../clientRoutes/qa'
```

Comment:
Is this being used? Can't tell if it is being used in the constructor, or if it is just remnants of previous code